packages/core/src/render3/di.ts

```diff
@@ -405,6 +405,14 @@ function lookupTokenUsingModuleInjector<T>(
 export function getOrCreateInjectable<T>(
     tNode: TDirectiveHostNode|null, lView: LView, token: ProviderToken<T>,
     flags: InjectFlags = InjectFlags.Default, notFoundValue?: any): T|null {
+  const embeddedInjector = lView[EMBEDDED_INJECTOR];
```

Comment:
nit: I think it'd be a bit more straightforward if we include `VIEW` into the name (so it's more obvious that this is an injector from an embedded view):
```suggestion
  const embeddedInjector = lView[EMBEDDED_VIEW_INJECTOR];
```